js/DataService.js

```diff
@@ -42,9 +41,19 @@ angular.module('PathOfDamage')
           description: "Modifiers that shift physical damage to elemental, typically reading like '% of Physical Damage taken as Y', such as Taste of Hate or Lightning Coil",
           tables: {
             shifts: {
-              totalName: "Damage Shifted",
-              totalMin: 0,
-              totalMax: 100
+              name: "Damage Shifted",
+              calcTotal: function () {
+               var totals = {};
+               var total = 0;
+                this.values.forEach(function (shift) {
+                  if (shift.enabled && shift.value && shift.element) {
+                    totals[shift.element] = totals[shift.element] + shift.value || shift.value;
+                    total += shift.value;
+                  }
+                });
+                this.totals = totals;
```

Comment:
yeah I originally had displayTotal, but didn't like that in the calculate method I have physDamage -= displayTotal. Perhaps naming totals to elementalTotals would just be better, although longer. It's somewhat moot since it will be revamped when the change the total displays, but probably better to be more clear in the meantime.